### PR TITLE
Adding words to en-devonfw.dict

### PR DESCRIPTION
**Pull request by ssarmokadam**

### CODE REVIEW

File: en-devonfw.dict [hunk 1]
```diff
@@ -1206,3 +1201,30 @@ jre
 propertymerge
 autocompletion
 Chocolatey
+
+FreeMarker
+JSON
+JUnit 
+Ionic
+IONIC
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:1209]: Which one is the correct spelling Ionic or IONIC? We should only have one.

File: en-devonfw.dict [hunk 2]
```diff
@@ -1206,3 +1201,30 @@ jre
...
+TS
+TypeScript
+POC
+IDE
+CMD
+SQL
+REST
+TCP
+URL
+csrf
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:1224]: It is an abbreviation so we should only have CSRF in the dict

File: en-devonfw.dict [hunk 3]
```diff
@@ -1206,3 +1201,30 @@ jre
...
+CSRF
+CLI
+SOAP
+WSDL
+EXT JS
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:1229]: What is this?

File: en-devonfw.dict [hunk 4]
```diff
@@ -1206,3 +1201,30 @@ jre
...
+HowTo's
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:1230]: Maybe only HowTo or is HowTo's needed for the spellcheck engine?

File: en-devonfw.dict [hunk 5]
```diff
@@ -17,7 +17,6 @@ JSR
 Balancer
 cdk
 Subclasses
-IServiceCollection
 util
 JTA
 ifdef
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:22]: Delete?

File: en-devonfw.dict [hunk 6]
```diff
@@ -47,7 +46,6 @@ CVEs
 QueryDSL's
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:46]: QueryDSL?

File: en-devonfw.dict [hunk 7]
```diff
@@ -70,9 +68,6 @@ WSL
 pdf
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:68]: PDF?

File: en-devonfw.dict [hunk 8]
```diff
@@ -1206,3 +1201,30 @@ jre
 propertymerge
```
Comment (SchettlerKoehler) [inline en-devonfw.dict:1201]: Delete?

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: en-devonfw.dict (hunk 5 above)
Comment (ssarmokadam) [reply en-devonfw.dict:22]: yes should be part covered in backticks part

File: en-devonfw.dict (hunk 8 above)
Comment (ssarmokadam) [reply en-devonfw.dict:1199]: These words are not handled in backticks etc should not be part of dict

File: en-devonfw.dict (hunk 3 above)
Comment (ssarmokadam) [reply en-devonfw.dict:1225]: Technical words which should not be detected as code

File: en-devonfw.dict (hunk 1 above)
Comment (ssarmokadam) [reply en-devonfw.dict]: Removed IONIC and just added Ionic to dict

File: en-devonfw.dict (hunk 7 above)
Comment (ssarmokadam) [reply en-devonfw.dict]: Added PDF

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**